docs/content/integrations/dbt/quickstart.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: "Quickstart: Run your dbt project with Dagster in 3 steps"
+description: Get started swiftly with this simple dbt + Dagster example.
+---
+
+# Quickstart: Run your dbt project with Dagster in 3 steps
+
+---
+
+## Step 1: Setup your environment
+
+<Note>
+  <strong>Note</strong>: We strongly recommend installing Dagster inside a
+  Python virtualenv. Visit the{" "}
+  <a href="/getting-started/install">Dagster installation docs</a> for more
+  info.
+</Note>
+
+Install dbt, Dagster, and the Dagster webserver/UI\*\*. Run the following to install everything using pip:
+
+```shell
+pip install dagster-dbt dagster-webserver
+```
+
+The `dagster-dbt` library installs both `dbt-core` and `dagster` as dependencies. Refer to the [dbt](https://docs.getdbt.com/dbt-cli/install/overview) and [Dagster](/getting-started/install) installation docs for more info.
+
+---
+
+## Step 2: Create your code location file
```

Comment:
It would be nice if we could model this how we've structured our other quickstart projects:

https://github.com/dagster-io/dagster/tree/master/examples/quickstart_snowflake

That way we can have a user scaffold the project using the template:

```
pip install dagster
dagster project from-example \
  --name my-dagster-project \
  --example quickstart_dbt
  ```

Reply:
That is what we did for our Dagster University dbt starter project:

https://github.com/dagster-io/dagster/tree/master/examples/project_du_dbt_starter

Reply:
yeah, I'd like to defer away from having users copy+paste large code snippets. ack that we have a history of it, but we can be better.

though I'm okay with not doing this until we un-experimentalize `DbtProject` and can merge/deprecate the existing dbt example project we have.